packages/compose/src/hooks/use-drop-zone/index.js

```diff
@@ -33,14 +33,14 @@ function useFreshRef( value ) {
 /**
  * A hook to facilitate drag and drop handling.
  *
- * @param {Object}                  props             Named parameters.
- * @param {boolean}                 props.isDisabled  Whether or not to disable the drop zone.
- * @param {(e: DragEvent) => void}  props.onDragStart Called when dragging has started.
- * @param {(e: DragEvent) => void}  props.onDragEnter Called when the zone is entered.
- * @param {(e: DragEvent) => void}  props.onDragOver  Called when the zone is moved within.
- * @param {(e: DragEvent) => void}  props.onDragLeave Called when the zone is left.
- * @param {(e: MouseEvent) => void} props.onDragEnd   Called when dragging has ended.
- * @param {(e: DragEvent) => void}  props.onDrop      Called when dropping in the zone.
+ * @param {Object}                  props               Named parameters.
+ * @param {boolean}                 [props.isDisabled]  Whether or not to disable the drop zone.
+ * @param {(e: DragEvent) => void}  [props.onDragStart] Called when dragging has started.
+ * @param {(e: DragEvent) => void}  [props.onDragEnter] Called when the zone is entered.
+ * @param {(e: DragEvent) => void}  [props.onDragOver]  Called when the zone is moved within.
+ * @param {(e: DragEvent) => void}  [props.onDragLeave] Called when the zone is left.
+ * @param {(e: MouseEvent) => void} [props.onDragEnd]   Called when dragging has ended.
+ * @param {(e: DragEvent) => void}  [props.onDrop]      Called when dropping in the zone.
```

Comment:
Marking these all as optional, based on actual use in the block-editor package like [here](https://github.com/WordPress/gutenberg/blob/604d8727978d131726df69ec0779cfaee698043b/packages/block-editor/src/components/use-block-drop-zone/index.js#L140) and [here](https://github.com/WordPress/gutenberg/blob/604d8727978d131726df69ec0779cfaee698043b/packages/block-editor/src/components/list-view/use-list-view-drop-zone.js#L259).

Code looks safe as well.